docs/operating-scylla/procedures/cluster-management/arbiter-dc.rst

```diff
@@ -0,0 +1,48 @@
+=========================================================
+Preventing Quorum Loss in Symmetrical Multi-DC Clusters
+=========================================================
+
+ScyllaDB requires at least a quorum (majority) of nodes in a cluster to be up
+and communicate with each other. A cluster that loses a quorum can handle reads
+and writes of user data, but cluster management operations, such as schema and
+topology updates are impossible.
+
+In clusters that are symmetrical, i.e., have an even number of datacenters (DCs)
+with the same number of nodes, losing a quorum may occur if one of the DCs
+becomes unavailable.
+For example, if one DC fails in a 2-DC cluster where each DC has three nodes,
+only three out of six nodes are available, and the quorum is lost.
+
+Adding another DC would mitigate the risk of losing a quorum, but it comes
+with network and storage costs. To prevent the quorum loss with minimum costs,
+you can configure an arbiter (tie-breaker) DC.
+
+An arbiter DC is a datacenter where all nodes are zero-token nodes -- nodes
+that don't replicate any data, but are only used for handling queries and Raft
+quorum voting. An arbiter DC maintains the cluster quorum if one of the other
+DCs fails, while it doesn't incur extra network and storage costs as it has no
+user data.
+
+Adding an Arbiter DC
+-----------------------
+
+To set up an arbiter DC, follow the procedure to
+:doc:`add a new datacenter to an existing cluster </operating-scylla/procedures/cluster-management/add-dc-to-existing-dc/>`.
+When editing the *scylla.yaml* file, set the ``join_ring`` parameter to
+``false`` on each node in that DC.
+
+Ensure that you set ``join_ring=false``:
+
+* On all nodes in the arbiter DC.
+* Before you start the nodes. If you set ``join_ring=false`` on a node that has
+  already been bootstrapped and owns a token range, the node startup will fail.
+  In such a case, you'll need to decommission the node, wipe it clean, and add
+  it back to the arbiter DC properly following the :doc:`procedure </operating-scylla/procedures/cluster-management/add-dc-to-existing-dc/>`.
```

Comment:
Looks great @annastuchlik ! I think we can reference our own docs for the revert procedure. 

So my final suggestion is: 

- Link `decommission` to https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/decommissioning-data-center.html
- Link `wipe it clean` to https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/clear-data.html

Cool stuff!